### PR TITLE
adding Woltka v0.1.6, pairends 

### LINKED ISSUE #3414: Move coverage.tgz and alignment.tar to be part of the per genome predictions
Currently, the coverage and alignment data are associated with the "Alignment profile" artifact, which contains a "none.biom" table. The features of the "none.biom" table are lineage strings, but the identifiers in coverage and the alignment data are the genome IDs. The feature table relevant to the coverage and alignment data are stored in a separate artifact "Per genome Predictions". 

It would be helpful if the coverage and alignment data were to co-located with the corresponding feature table.

### PR DESCRIPTION
**Pull request by antgonza**
… mux/demux

### CODE REVIEW

File: scripts/woltka_merge [hunk 1]
```diff
@@ -1,66 +1,40 @@
 #!/usr/bin/env python
 
 import os
-import biom
-import glob as glob_
-import h5py
+# import biom
```
Comment (charles-cowart) [inline scripts/woltka_merge]: should these commented imports be removed?

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything appears in order. Just one question.

**Comment by charles-cowart** [comment]
just to clarify, we are consulting with @wasade  on one small point before final approval and merging.

**Comment by antgonza** [comment]
@charles-cowart, this is ready for final review & merge.

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**